monitor/alerts/malformed.py

```diff
@@ -1,13 +1,22 @@
 import json
-from dataclasses import dataclass
+from typing import List, Optional
 
 from clients.slack.schema import SlackMessageSchema
 from monitor.alerts.alert import Alert
 
 
-@dataclass
 class MalformedAlert(Alert):
-    data: dict
+    def __init__(
```

Comment:
Convert from dataclass into class.
1. IMO dataclass should be used in very specific situations when the class is proper dataclass (and usually it will be better to use pydantic schema)
2. dataclass and inheritance doesn't work so well, and we inheritance (with default values) this class.